features/toolbar/components/ToolbarContainer.native.js

```diff
@@ -22,25 +23,40 @@ export class ToolbarContainer extends Component {
     render() {
         var underlayColor = ColorPalette.buttonUnderlay;
         var micButtonStyle;
+        var micIconStyle;
         var micButtonIcon;
         if (this.props.audioMuted) {
             micButtonStyle = [
                 styles.toolbarButton,
                 { backgroundColor: underlayColor }
             ];
-            micButtonIcon = 'microphone-slash';
+            micIconStyle = [styles.icon, { color: 'white' }];
+            micButtonIcon = 'mic-disabled';
         }
         else {
             micButtonStyle = styles.toolbarButton;
+            micIconStyle = styles.icon;
             micButtonIcon = 'microphone';
         }
 
+        // The following property is responsible to hide/show the toolbar
+        // view by moving it out of site of the screen boundaries.
+        // An attempt to use the opacity property has been made in order
+        // to eventually implement a fadeIn/fadeOut animation, however a known
+        // react native problem has been doscovered, which allows the view to
+        // still capture touch events even if hidden. Alternatives will be
+        // investigated.
+        var bottomValue = styles.toolbarContainer.bottom;
+        if (!this.props.isVisible)
+            bottomValue = -Dimensions.get('window').height;
```

Comment:
The following action item in my comment above:
"Avoid duplication and provide a Hide-ableView abstraction, which can also implement animation."
was meant for exactly this code duplication. A separate PR will take care of creating an abstraction and providing a clean reusable view that allows hiding with animation.